pswi/PSWI-marist.sh

```diff
@@ -0,0 +1,113 @@
+export ZOSMF_URL="https://zzow03.zowe.marist.cloud"
+export ZOSMF_PORT=10443
+export ZOSMF_SYSTEM="S0W1"
+export DIR="/u/zowead2"
+export CSIHLQ="ZWE.PSWI.AZWE001"
+export ZONE="TZONE"
+export TMP_ZFS="ZOWEAD2.TMP.ZFS"
+export ZOWE_ZFS="${CSIHLQ}.ZFS"
+export ZOWE_MOUNT="/u/zwe/zowe-smpe/"
+export VOLUME="ZOS003"
+export TEST_HLQ="ZOWEAD2.PSWIT"
+export VERSION="1.25.0"
+export SYSAFF=2964 
+export ACCOUNT=1
+
+# Variables for workflows
+# SMPE
+export SMPMCS="ZOWEAD2"
+export FMID="AZWE001"
+export RFDSNPFX="ZOWE"
+export CSIVOL="ZOS003"
+export TZONE=$ZONE
+# CSIHLQ for workflow is same as for PSWI
+export DZONE="DZONE"
+export THLQ="${CSIHLQ}.T"
+export DHLQ="${CSIHLQ}.D"
+export TVOL=$CSIVOL
+export DVOL=$CSIVOL
+export MOUNTPATH=$ZOWE_MOUNT
+#PTF
+export CSI=$CSIHLQ
+export PTFDATASET="ZOWEAD2.ZOWE.AZWE001"
+export TARGET=$TZONE
+export DISTRIBUTION=$DZONE
+export PTF1="UO01994"
+export PTF2="UO01995"
```

Comment:
I have a question, is this hardcoded to always build PSWI on top of Zowe v1.25.0?

Do you think it makes sense to dynamically pick up the SMPE build from previous stage?

Reply:
Yes, it is hardcoded there as we are now not downloading anything. With the previous stage is problem that it is not creating classic SMP/E environment. Which we are now creating here by our workflows from datasets that are already on zzow3. Same with the application of PTFs, the datasets are there.